VimeoNetworkingExample-iOS/VimeoNetworkingCommonTests/VIMVideoTests.swift

```diff
@@ -28,11 +28,11 @@ import XCTest
 
 class VIMVideoTests: XCTestCase
 {
+    private var liveDictionary: [String: Any] = ["link": "vimeo.com", "key": "abcdefg", "activeTime": Date(), "endedTime": Date(), "archivedTime": Date()]
```

Comment:
Can we reset this dictionary to have those key-value pairs in `tearDown` so that when new tests whose implementations do not modify the value in the `status` key are introduced, they can start with the original key-value pairs?

Reply:
Properties like this get reset on every test already, so `liveDictionary` will always start out in this state. As far as I understand it altering the property in `tearDown()`, either the class or instance method, wouldn't have any effect here. 

This can be verified by setting breakpoints and inspecting `liveDictionary` at the start of each test. The tests would also likely fail if the dictionary weren't already being reset for every test.

Reply:
I didn't realize this. I thought that setup gets called before each test and that's the place to re-instantiate things.